<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSystemConfigDlg</class>
 <widget class="QDialog" name="QSystemConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSystemConfigDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>系统设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" rowspan="2" colspan="3">
       <widget class="QLabel" name="label_version">
        <property name="text">
         <string>版本：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_SetDiscBinType">
        <property name="title">
         <string>设置光盘桶类型</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>光盘桶1：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_bin1"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>光盘桶2：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_bin2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>光盘桶3：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_bin3"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>光盘桶4：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_bin4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_getsysconfig">
        <property name="text">
         <string>获取设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
